docs/COMPONENT_INTEGRATION.md

```diff
@@ -210,7 +210,7 @@ func (s *componentHandler) GetManagementState(dsc *dscv1.DataScienceCluster) ope
 
 func (s *componentHandler) NewCRObject(dsc *dscv1.DataScienceCluster) common.PlatformObject
 
-func (s *componentHandler) Init(platform cluster.Platform) error 
+func (s *componentHandler) Init(platform common.Platform) error 
```

Comment:
This is probably out of scope for this JIRA but I noticed the component-codegen CLI [template](http://github.com/opendatahub-io/opendatahub-operator/blob/main/cmd/component-codegen/templates/component_handler.go.tmpl#L58) still has `cluster.Platform` and not `common.Platform`.

Is it intended to be `cluster.Platform`? Should the template be changed as well?

Reply:
agreed!

Reply:
maybe a separate pr would be good 👍  Dont want to pollute this one

Reply:
Moved docs and template update to [separate PR](https://github.com/opendatahub-io/opendatahub-operator/pull/1770)